### ISSUE TITLE
calculation of grpo loss

### ISSUE THREAD

**Issue by QuLiao1117** [CaraJ7/T2I-R1#11]
https://github.com/CaraJ7/T2I-R1/blob/7fe2d2783df158700328ac652d87705ff58b92fe/src/t2i-r1/src/open_r1/trainer/grpo_trainer.py#L767

Hi, thanks for sharing this great work! May I ask why using `per_token_logps - per_token_logps.detach()` instead of `per_token_logps - ref_per_token_logps.detach()` in this line? Would per_token_logps - per_token_logps.detach() results in 0?
